src/app/static/src/app/components/downloadResults/Download.vue

```diff
@@ -8,17 +8,17 @@
             <download-icon size="20" class="icon ml-2" style="margin-top: -4px;"></download-icon>
         </button>
         <div>
-            <download-progress v-if="!modalOpen" id="progress"
```

Comment:
This `v-if` condition was because previously, setting off an upload via the upload modal would first trigger the outputs to be prepared (even if already existent!) and so without this condition this progress bar would appear underneath the download buttons. Now that the logic has been simplified, this will not happen.